src/ext/client-side-templates.js

```diff
@@ -1,9 +1,9 @@
 htmx.defineExtension('client-side-templates', {
     transformResponse : function(text, xhr, elt) {
+        var data = JSON.parse(text);
```

Comment:
I don't want to parse the JSON unless we know for sure there is a client side template associated with this request.  The hx-ext="client-side-templates" attribute can live on a parent tag (e.g. the body) so we could end up trying to parse non-JSON responses as JSON, throwing an exception

Reply:
AH that's fair, sounds good